docs/developer.md

```diff
@@ -0,0 +1,189 @@
+# Developer documentation
+
+## On-demand runs in the stage environment
+This trick might be useful during the development phase. In case you want to
+check if your changes are working, before submitting a PR and having it merged
+then automatically built by Konflux, it will be useful to have these on-demand
+runs. This is not a mandatory step, so feel free to skip it.
+
+Follow these steps to make an on-demand run in the stage environment:
+- login to one of the stage clusters listed in
+  https://eisraeli.pages.redhat.com/rhtap-link/#ocp-clusters (use RH SSO)
```

Comment:
This is a public repo, I think we don't want to share this link here, since it's RH only.